option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message CommonRankLikeRequest
{
 required uint32 type = 1;
 required U64Type like_player_id = 2;
}
